### ISSUE TITLE
Running on runpod.io errors out

### ISSUE THREAD

**Issue by maxfridbe** [ashleykleynhans/audiocraft-docker#4]
tried it on 3 pods, but maybe im missing something
```
UnboundLocalError: local variable 'sentencepiece_model_pb2' referenced before assignment
Loading model GrandaddyShmax/musicgen-large
Traceback (most recent call last):
  File "/workspace/venv/lib/python3.10/site-packages/gradio/routes.py", line 488, in run_predict
    output = await app.get_blocks().process_api(
  File "/workspace/venv/lib/python3.10/site-packages/gradio/blocks.py", line 1431, in process_api
    result = await self.call_function(
  File "/workspace/venv/lib/python3.10/site-packages/gradio/blocks.py", line 1109, in call_function
    prediction = await anyio.to_thread.run_sync(
  File "/workspace/venv/lib/python3.10/site-packages/anyio/to_thread.py", line 33, in run_sync
    return await get_asynclib().run_sync_in_worker_thread(
  File "/workspace/venv/lib/python3.10/site-packages/anyio/_backends/_asyncio.
```

**Comment by ashleykleynhans** [comment]
Thanks for pointing this out, nobody has made me aware of this issue.
Seems the latest transformers versions need `protobuf` to be installed.
I am currently building a new Docker image to resolve the issue,

**Comment by ashleykleynhans** [comment]
Its fixed now in version 3.0.3 of the template, thank you for reporting it.
